components/compliance-service/inspec-agent/resolver/resolver.go

```diff
@@ -125,10 +126,11 @@ func (r *Resolver) handleAzureApiNodes(ctx context.Context, m *manager.NodeManag
 				logrus.Debugf("handleManagerNodes unable to get node id: %s", err.Error())
 			}
 			if len(nodeUUID) == 0 {
-				nodeUUID, err = r.scannerServer.AddNodeToDB(ctx, &inspec.OSInfo{}, &baseJob, scanner.SourceInfo{SourceAccountID: m.AccountId, SourceRegion: node.ID, SourceID: node.ID}, time.Time{})
+				id, err := uuid.NewV4()
```

Comment:
question for myself: do we need to do anything about creating a uuid here? why? won't it get done later in `ProcessNode`?

Reply:
answer: if there's no node uuid, the report will get rejected! so we need this